### DISCUSSION TITLE
Internationalization (i18n) and localization (l10n)

### DISCUSSION THREAD

**Discussion by sphakka** [lldap/lldap#880] [Q&A] [answered]
I just started playing around with lldap for a small French-speaking user base and realized that, unless missing something, no translations are available. 

Is lldap i18n/l10n-ready? If so, I'd contribute some translations, just please give advice on how to proceed ^^

**Comment by nitnelave** [comment, marked as answer]
See https://github.com/lldap/lldap/issues/20 (very old issue!)
In short, it'd a lot of effort for not much gained.

**Reply by sphakka** [reply]
Thanks for the bug reference -- it escaped my searches, indeed.  

Given that people are _ready_ to contribute, I'd really not dismiss this idea so quickly: once bootstrapped, it will easily progress and the project will gain popularity!

**Reply by nitnelave** [reply]
I'm not so worried about the translations, but more about the bootstrapping process: it's not _so_ easy to put in place, and the development resources (well, my time) are _very_ limited these days.
